programs/hh-escrow/src/state/uri.rs

```diff
@@ -2,11 +2,10 @@ use anchor_lang::prelude::*;
 
 use crate::error::ErrorCode;
 
-const URI_MAX_LEN: usize = 200;
+// FIXME: This should be 200;
+const URI_MAX_LEN: usize = 256;
```

Comment:
Is the comment above meant to be addressed in current PR?

Reply:
No, it's here as a reminder of one of the things that needs fixing.